README.md

```diff
@@ -25,21 +24,13 @@ Exercism exercises in Haskell
 - [Running tests](#running-tests)
 - [Running HLint](#running-hlint)
 
-### Asking for help
-When in doubt just ask!
-- [gitter support](https://gitter.im/exercism/support): more specific/technical questions.
-- [gitter dev](https://gitter.im/exercism/dev): technical questions.
```

Comment:
empirically, looking at all instances of the world "Haskell" occurring in exercism/dev dating back to January 2016, I count exactly one instance where it was a contributor looking to contribute. So at the rate of one contributor every 3.75 years, It seems we don't lose much by removing this link. Having them write an issue seems to work fine.

I have no data on exercism/support since I'm not interested in that channel.